native_client/java/app/src/main/java/deepspeech/mozilla/org/deepspeech/DeepSpeechActivity.java

```diff
@@ -38,8 +38,8 @@
     final int N_CEP = 26;
     final int N_CONTEXT = 9;
     final int BEAM_WIDTH = 50;
-    final float LM_WEIGHT = 1.50f;
-    final float VALID_WORD_COUNT_WEIGHT = 2.10f;
+    final float LM_ALPHA = 0.75f;
+    final float LM_BETA = 1.85f;
```

Comment:
Dumb question, but where is this value used?

Reply:
I don't think it is at the moment. As far as I can tell the Java demo is not currenlty using the LM.